src/components/ErrorBoundary/index.module.css

```diff
@@ -0,0 +1,45 @@
+.boundary {
+  background: var(--blue);
+  background: linear-gradient(
+    to right,
+    #1e5799 0%,
+    #207cca 34%,
+    #2989d8 71%,
+    #7db9e8 100%
+  );
+
+  width: 100%;
+  height: 100%;
+}
+.boundary-wrapper {
+	max-width: 768px;
```

Comment:
Just a little code culture point. It's beautiful and readable, when all lines have the same space from the left. Alignment is a view of system, it's like a music in the code.

It's ok if you want to play some jazz. But maybe we do it more classical for other guys? )))

Reply:
Okay, got it.